adafruit_pybadger.py

```diff
@@ -103,6 +103,7 @@ def __init__(self, i2c=None):
 
         # Display
         self.display = board.DISPLAY
+        self._brightness_value = 1.0
```

Comment:
This appears to be a value for display brightness. Can we name it something like _display_brightness so it doesn't get confused with neopixel brightness? It makes reading the code a bit easier. Thanks

Reply:
Excellent idea. I'll make the change.